src/cloud/cdn/fastly-vcl-badreferer.md

```diff
@@ -59,43 +59,41 @@ For more information about Edge Dictionaries, see [Creating and using Edge Dicti
 
 ## Create a custom VCL snippet to block referrer spam
 
-The following custom VCL snippet code (JSON format) checks incoming requests and blocks requests from any referrer site included in the `referrer_blocklist` edge dictionary.
+The following custom VCL snippet code (JSON format) shows the logic to check and block requests. The VCL snippet captures the host of a referrer website into a header, and then compares the host name to the list of URLs in the `referrer_blocklist` dictionary. If the host name matches, the request is blocked with a `403 Forbidden` error.
 
 ```json
 {
   "name": "block_bad_referrer",
   "dynamic": "0",
   "type": "recv",
   "priority": "5",
-  "content": "set req.http.Referer-Host = regsub(req.http.Referer, \"^https?://?([^:/\\s]+).*$\", \"\\1\"); if (table.lookup(referrer_blocklist, req.http.Referer-Host)) { error 403 \"Forbidden\"; }"
+  "content": "set req.http.Referer-Host = regsub(req.http.Referer, "^https?://?([^:/s]+).*$", "\1"); if (table.lookup(referrer_blocklist, req.http.Referer-Host)) { error 403 "Forbidden"; }"
```

Comment:
Looks like there are some formatting problems here, possibly caused by markdown literals. It may be just GitHub, but should double check the contents of the code block.